client/src/javascript/components/modals/settings-modal/SettingsModal.js

```diff
@@ -220,7 +221,14 @@ class SettingsModal extends React.Component {
           onSettingsChange: this.handleFloodSettingsChange,
           scrollContainer: this.modalBodyRef
         }
-      }
+      },
+      about: {
+          content: AboutTab,
+          label: this.props.intl.formatMessage({
+              id: 'settings.tabs.about',
+              defaultMessage: 'About'
+          })
```

Comment:
There should be an eslint rule for this, but let's use 2 characters for tab width please.